packages/atomic/src/components/atomic-modal/atomic-modal.pcss

```diff
@@ -0,0 +1,88 @@
+@import '../../global/global.pcss';
+
+[part='backdrop'] {
+  pointer-events: none;
+
+  @screen desktop-only {
+    display: grid;
+    place-items: center;
+    grid-template-areas:
+      '. .     .'
+      '. modal .'
+      '. .     .';
+    grid-template-columns: 1fr auto 1fr;
+    grid-template-rows: 1fr auto 3fr;
```

Comment:
When there's space available vertically, the modal keeps 3 times as much empty space below itself as above. This matches Johnny's design for the smart snippet modal on desktop.